core/modules/system/css/system.admin.css

```diff
@@ -221,6 +221,43 @@ span.admin-missing {
 .system-status-report .report-error .status-summary {
   color: #EC351C;
 }
+
+.system-status-report .status-description .item-list h3 {
+  font-size: 1em;
+}
+.system-status-report ul.requirements {
+  margin-left: 0; /* LTR */
+  padding-left: 0; /* LTR */
+}
+[dir="rtl"] .system-status-report ul.requirements {
+  margin-right: 0;
+  padding-right: 0;
+}
+.system-status-report ul.requirements li.status-ok,
+.system-status-report ul.requirements li.status-error {
+  list-style: none;
+  padding-left: 1em; /* LTR */
+  text-indent: -2ch;
+}
+[dir="rtl"] .system-status-report ul.requirements li.status-ok,
+[dir="rtl"] .system-status-report ul.requirements li.status-error {
+  padding-right: 1em;
+}
+.system-status-report ul.requirements li.status-ok:before {
+  content: "✓";
+  color: green;
+  padding-right: 1ch; /* LTR */
+}
+.system-status-report ul.requirements li.status-error:before {
+  content: "✗";
```

Comment:
Tiny accessibility nit-pick: contrast to the light grey background is only 3.73 (fails AA).

As far as I know, screen readers do read CSS content - not sure what they read for these characters, and if it's useful or distracting.

Reply:
Good point @indigoxela 👍🏼 ...working on it.

Reply:
...I've checked `#ad0000` and `#006100` against all possible table row colors in the Seven admin theme:
- `#ffffff` (odd table rows)
- `#f7f7f7` (even table rows)
- `#ffff99` (hovered-over rows)

Reply:
...I've also used ChromeVox on my Mac, and it read the items properly  as:
`List with 6 items - GIF image format supported - List item - JPEG image format supported - List item ...`

So I think we are good re screen readers 👍🏼 

PS: while using the screen reader, I spotted may other issues with the status report page and tables in general, but that's a larger issue that should be dealt with elsewhere - we are NOT making things worse with this PR here.